App.js

```diff
@@ -1,25 +1,37 @@
 import React from 'react';
-import styled from 'styled-components/native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
 
-const Container = styled.View`
-	flex: 1;
-	background-color: papayawhip;
-	justify-content: center;
-	align-items: center;
-`;
+import HomePage from './components/HomePage';
+import Movies from './components/Movies';
 
-const Title = styled.Text`
-	font-size: 24px;
-	color: palevioletred;
-`;
+const Tab = createMaterialTopTabNavigator();
 
 const App = () => {
+
 	return (
-		<Container>
-			<Title>This is your cool app!</Title>
-			<Title>Go to App.js and start coding</Title>
-			<Title>💅💅💅</Title>
-		</Container>
+		<NavigationContainer>
```

Comment:
Great navigation. And nice that the swipe was built in. I began with the tabBarBottom and that one didn't have swipe function. So I changed :)